src/reducers/VaccineReducer.js

```diff
@@ -0,0 +1,39 @@
+import { VACCINE_ACTIONS } from '../actions/VaccineActions';
+
+const initialState = () => ({
+  scannedSensors: [],
+  isScanning: false,
+});
+
+export const VaccineReducer = (state = initialState(), action) => {
+  const { type } = action;
+
+  switch (type) {
+    case VACCINE_ACTIONS.SCAN_START: {
+      return {
+        ...state,
+        isScanning: true,
+      };
+    }
+    case VACCINE_ACTIONS.SCAN_STOP: {
+      return {
+        ...state,
+        isScanning: false,
+      };
+    }
+    case VACCINE_ACTIONS.SENSOR_FOUND: {
+      const { payload } = action;
+      const { macAddress } = payload;
+
+      const { scannedSensors } = state;
+      scannedSensors.push(macAddress);
```

Comment:
perhaps `sensorAddresses`? though that feels a little clumsy to me. I'm just wondering at the equivalence of a mac address and a 'sensor' in the naming of the collection

Reply:
Oops, I agree it's a little misleading as we are not really storing sensors 😆 . I tried `scannedSensorAddresses`? Too long? 8dbd7ef0